domain/src/main/java/org/oppia/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProvider.kt

```diff
@@ -12,7 +12,7 @@ import javax.inject.Inject
  *
  * https://github.com/oppia/oppia/blob/37285a/extensions/interactions/TextInput/directives/text-input-rules.service.ts#L24
  */
-internal class TextInputEqualsRuleClassifierProvider @Inject constructor(
+public class TextInputEqualsRuleClassifierProvider @Inject constructor(
```

Comment:
I think here public comes in redundant visibility modifier, which I don't think we need it. WDYT?

Reply:
You are very right! Thank you! Changing this now :)